File: exercicios_beecrownd/1116.py
n = int(input())
count = 0

while count < n:
    x, y = [int(x) for x in input().split()]
    if y == 0:
        print("divisao impossivel")
    else:
        div = float(x/y)
        print(f"{div:.1f}")
    
    count+=1

File: exercicios_beecrownd/1116.c
#include <stdio.h>

int main(){

    int n, count =0, x, y;
    float div;
    scanf("%d", &n);

    while(count<n){

        scanf("%d %d", &x, &y);
         count++;
        if(y==0){
            printf("divisao impossivel\n");
        }
        else if(x==0){
            div = 0.0;
            printf("%.1f\n", div);
        }
        else{
           
            div = ((float)x/y);
            printf("%.1f\n", div);
        }
    


    }

    return 0;
}
